### ISSUE TITLE
Support for -l / --long

### ISSUE THREAD

**Issue by allenye0119** [athityakumar/colorls#40]
Is it possible to support the `-l` option to behave like `ls -l`?

**Comment by athityakumar** [comment]
Hey @allenye0119! Thanks for filing this issue. If this is required, the only way might be to workaround by parsing the output of `ls -l` using backticks in Ruby (or similar system calls). Keeping this in mind, do you think `-l` / `--list` flag needs to be supported?

**Comment by allenye0119** [comment]
I think the information provided by `-l / --list` is really helpful, e.g. file permission, file size (when combined with `-h / --human-readable`) and the symlink destination.

**Comment by athityakumar** [comment]
Hmm, not sure how well it fits in with the other flags. But, it'd definitely fit well as a stand-alone flag.

If anyone likes to work on this issue, please refer to this approach (psuedocode-ish) I haveve in my mind. 😄 

```ruby
output = `ls -l #{input}`
arrays = output.split("\n").map { |line| line.split(' ') }

puts arrays.map do |line|
   line.map do |word|
       # check if word is a file / directory
       # and change it appropriately
       # with (color + icon) or just white.
   end.join("\t")
end.join("\n")
```

**Comment by kaustubhhiware** [comment]
I think this was debated at the very start as well. Initially, `lc -l` was supported, relevant discussion at #4

**Comment by GladOSkar** [comment]
I would like to work on this.
I am no expert but rather than parsing ls (which is apparently [discouraged](http://mywiki.wooledge.org/ParsingLs)) i would recommend printing all info from `File.stat(@input)` since it is all given there.
What do you think?

**Comment by athityakumar** [comment]
@GladOSkar - Yes, I knew that ls parsing is a discouraged approach. But, I wasn't aware of `File#stat` method. Sure, go ahead with it. I'm assigning this to you. 😄

**Comment by craftzdog** [comment]
+1

**Comment by athityakumar** [comment]
Fixed with PR #63. Closing this issue now. Good work, @GladOSkar 👍
